action.yml

```diff
@@ -254,7 +240,7 @@ runs:
         echo "CA_GH_ACTION_TARGET_PORT=${CA_GH_ACTION_TARGET_PORT}" >> $GITHUB_ENV
 
     - name: Determine default target port if not provided and no Dockerfile provided/found
-      if: ${{ inputs.targetPort == '' && env.CA_GH_ACTION_DOCKERFILE_PATH == '' && inputs.imageToDeploy == '' }}
+      if: ${{ inputs.targetPort == '' && env.CA_GH_ACTION_DOCKERFILE_PATH == '' && inputs.appSourcePath != '' }}
```

Comment:
Could you please provide more context about this change?

Reply:
Sure thing -- I wanted to more explicit about what dictates the "builder scenario", and in our case, it's when the user provides a value for `appSourcePath` (their application source) and there's no assigned value to the environment variable `CA_GH_ACTION_DOCKERFILE_PATH` (either the `dockerfilePath` argument provided by the user or the `Dockerfile` we found in their application source).

This step is attempting to figure out the target port that the Container App should listen on if one wasn't provided, and it does so by taking the `runtimeStack` that was either provided or detected by us and choosing `80` or `8080` depending on the platform provided/detected. Since the `runtimeStack` is _only_ used in the builder case, this step should only be used when the two values mentioned above meet the conditions specified, and is independent of the value of `imageToDeploy`.